README.md

```diff
@@ -63,71 +119,129 @@ The following states are used to represent the FT's service tiers:
+</time>
+```
+
+As with the indicator label, we recommend using [o-date](https://registry.origami.ft.com/components/o-date) to format the timestamp element.
+
+## Sass
+
+Output all label styles with the `oLabels` mixin:
 
 ```scss
 @include oLabels();
 ```
 
-```css
-.o-labels {
-	/* styles */
-}
-.o-labels--big {
-	/* styles */
-}
-/* etc. */
-```
+Or pass an options map `$opts` argument to output just the label styles you need. Options include:
 
-If you wish to specify a subset of sizes and states to output styles for, you can pass in options (see [sizes](#sizes) and [states](#states) for available options):
+- sizes: a list of standard label sizes to output (see [available sizes](#standard-label-sizes))
+- states: a list of standard label states to output (see [available states](#standard-label-states)))
+- indicators
+	- status: a list of indicator label statuses to output (see [available statuses](#indicator-label-status))
+	- inverse
+- timestamp
+	- inverse
 
 ```scss
 @include oLabels($opts: (
+	// standard label sizes to output
+	// e.g. .o-labels--big
 	'sizes': ('big'),
+	// standard label states to output
+	// e.g. .o-labels--content-commercial
 	'states': (
 		'content-commercial',
 		'content-premium'
+	),
+	// indicator label labels to output
+	// .o-labels-indicator
+	'indicators': (
+		// indicator label statuses to output
+		// .o-labels-indicator--live
+		'status': ('live'),
+		// whether to output the indicator label inverse style
+		// .o-labels-indicator--inverse
+		'inverse': true,
+	),
+	// output the timestamp label label
+	// .o-labels-timestamp
+	'timestamp': (
+		// whether to output the timestamp label inverse style
+		// .o-labels-timestamp--inverse
+		'inverse': true
 	)
 ));
 ```
 
-### Mixin: `oLabelsAddState`
+### Custom Standard Label State
 
-The `oLabelsAddState` mixin can be used to output a class for one of the label states, outlined in the [states table](#states):
-
-```scss
-@include oLabelsAddState('content-commercial');
-```
-
-```css
-.o-labels--content-commercial {
-	/* styles */
-}
-```
-
-The `oLabelsAddState` mixin also accepts optional custom configurations, which override defaults or allow you to define your own label states:
+Use `oLabelsAddState` mixin to add a custom label state for the standard label. See the [`oLabelsAddState` SassDoc](https://registry.origami.ft.com/components/o-labels/sassdoc?brand=master#mixin-olabelsaddstate) for more details.
 
 ```scss
+// outputs a class .o-labels--citrus-fruit
 @include oLabelsAddState('citrus-fruit', (
 	background-color: oColorsByName('lemon')
 ));
 ```
 
-```css
-.o-labels--citrus-fruit {
-	/* styles */
-}
-```
+### Custom Markup
 
-### Mixin: `oLabelsContent`
+When it's not possible to use an `o-labels` CSS class, for example within another Origami component, use:
+- `oLabelsContent` to output a standard label with a custom class.
+- `oLabelsIndicatorContent` to output an indicator label with a custom class.
```

Comment:
OH! Can we use `oLabelsIndicatorContent` to create our own custom indicator?

Reply:
thanks for flagging I forgot about `closed`! :) It might be a little preemptive given its only used in live blogs at the moment but I think I'll add a `closed` variant. It'll allow us the flexibility of changing the style of the closed status independently of other statuses and allow us to avoid using the `oLabelsIndicatorContent` api, which is a bit cumbersome.

I've seen references to a `comingsoon` label too. After chatting with Josh we figured `comingsoon` is not used and I haven't included it. Do you think that's right?

Reply:
Yup, we don't use the `comingsoon` label.